sdk/aqueduct/tests/decorator_test.py

```diff
@@ -24,7 +24,6 @@ def test_packaging_files():
 
         assert not os.path.exists("./test_function/test_function.py")
         assert os.path.exists("./test_function/test_dependency_folder/helper_function.py")
-        assert os.path.exists("./test_function/requirements.txt")
```

Comment:
Out of the loop on this one--why do we no longer have `requirements.txt`? Is this part of this PR?

Reply:
oh yeah at lot of the context is here: https://linear.app/aqueducthq/issue/ENG-1191/remove-pipreqsnb-for-requirements-discovery

Essentially, we wanted to cut out pipreqsnb because of its weirdness. We will still copy requirements.txt into the zip file if it exists in the same directory as the function - we just won't infer.